### PR TITLE
Change build SVG to Azure DevOps

### PR DESCRIPTION
**Pull request by jiasli**
As we have moved from Travis CI to Azure DevOps, change the build SVG accordingly to use Azure DevOps's [Status - Get](https://docs.microsoft.com/en-us/rest/api/azure/devops/build/status/get?view=azure-devops-rest-6.0) API.

To get the URLs, open https://dev.azure.com/azure-sdk/public/_build?definitionId=1643&_a=summary and click

image

Output:

```
[![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/cli/microsoft.knack?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=1643&branchName=master)
```

### CODE REVIEW

**Review by fengzhou-msft — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -7,8 +7,8 @@ Knack
 .. image:: https://img.shields.io/pypi/pyversions/knack.svg
     :target: https://pypi.python.org/pypi/knack
 
-.. image:: https://travis-ci.org/Microsoft/knack.svg?branch=master
-    :target: https://travis-ci.org/Microsoft/knack
+.. image:: https://dev.azure.com/azure-sdk/public/_apis/build/status/cli/microsoft.knack?branchName=master
+    :target: https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=1643&branchName=master
```
Comment (evelyn-ys) [inline README.rst]: `branchName` is still `master` instead of changing to `dev`?

Comment (jiasli) [reply README.rst]: Kindly refresh it.

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**